{"name": "测试03用例名称", "status": "passed", "start": 1611064674272, "stop": 1611064674272, "uuid": "070f428d-3185-4c22-9edf-ebe0ce2eff56", "historyId": "aaeffdda95d4d34ff4d03238f03c8139", "testCaseId": "e69660cb497d57b4d0abb880f6116dec", "fullName": "test_case03#test_03", "labels": [{"name": "story", "value": "用户下单失败的情况"}, {"name": "suite", "value": "test_case03"}, {"name": "host", "value": "DESKTOP-LTNMVII"}, {"name": "thread", "value": "4580-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case03"}]}